### DISCUSSION TITLE
Action Bar - https://fundamental-ngx.netlify.app/#/core/action-bar

### DISCUSSION THREAD

**Discussion by kateyusap** [SAP/fundamental-ngx#9563] [General]
Hi team,

Good afternoon.
My name is Kate Yu working at the BN UX design team. 
I have a question about NGX/Angular. Is the team working on the style from the central library guideline? I noticed the action bar buttons are primary on the right and secondary on the left. And the page title and subtitle should be 8px between. But on the NGX. It has no space. Please share your based stand design point with me. I want to use it as my product design standard. And Based on our team use case, we have pagination updates that want to chat with your team. You could consider adding it to the library.

My email is kate.yu@sap.com. I am looking forward to hearing from you.

Thank you for your hard work.

Best,
Kate

**Comment by droshev** [comment]
Hi @kateyusap we are discussing the design of the action bar with some designers. Once they come up with an official design we (may need to) update the implementation

**Comment by kateyusap** [comment]
Thank you, @droshev, for the information. Can you meet with me to discuss some components that show the differences between NGX and the brand library? Thank you.
